src/components/common/AppHeader.vue

```diff
@@ -52,12 +52,19 @@
               v-if="session.experimentalMode"
               :to="{ name: 'notifications' }"
             >
-              <i class="material-icons">notifications</i>
+              <img
+                v-if="isNotificationAvailable"
+                class="notification-bell"
+                src="/img/icons/notifications/bell-icon-alert.png"
+                alt="bell icon"
```

Comment:
```suggestion
                alt="bell icon alert"
```

Reply:
wait wait we resolved to use load 2 png's instead of a material icon and a css circle? this doesn't seem right.

Reply:
this is how extension works